components/Chat/Message/MessageReactionsList.tsx

```diff
@@ -290,32 +293,23 @@ const useStyles = () => {
       height: 119.5,
     },
     itemText: {
-      color: textSecondaryColor(colorScheme),
+      color: theme.colors.text.primary,
       fontSize: 16,
       lineHeight: 20,
```

Comment:
should not need this since default preset have those properties

Reply:
removed, just kept the margin!

Reply:
Perfect! No margin should never be in design-system elements IMO 😮

Reply:
Ok, will review this :) thanks!